bff-nav_ansatt/src/main/kotlin/no/nav/amt/tiltak/bff/nav_ansatt/AutentiseringController.kt

```diff
@@ -17,43 +13,22 @@ import org.springframework.web.bind.annotation.RestController
 open class AutentiseringController(
     private val authService: AuthService,
     private val navAnsattService: NavAnsattService,
-	private val adGruppeService: AdGruppeService
 ) {
 
 	@ProtectedWithClaims(issuer = Issuer.AZURE_AD)
 	@GetMapping("/meg")
 	fun me(): MegDto {
```

Comment:
Litt usikker på om det å fjerne tilganger fra `/meg` er det vi ønsker? Men jeg ser ikke noen stor grunn til å fortsette med det, ettersom vi kaster 403 på autoriseringssjekkene om de ikke har tilganger.

Reply:
Tror ikke det lønner seg å fjerne det iom at det er forventet oppførsel at det skal gjøres tilgangskontroll på flate og endringsmeldinger separat. I frontend er det sjekk på dette nå. Har det evt vært en diskusjon på det som jeg ikke har fått med meg? 
Man kan fortsette å returnere tilganger men at de settes basert på hva som ligger på tokenet isteden for poao-tilgang.

Reply:
Tilgangskontrollen på flate og endringsmeldinger er fortsatt separat. Jeg ser ikke helt vinsten med å opprettholde dette. Vi brukte uansett kun tilgangskontroll på endringsmeldinger via `/meg`.